projects/storefrontapp-e2e-cypress/cypress.json

```diff
@@ -6,6 +6,11 @@
   "env": {
     "CLIENT_ID": "mobile_android",
     "CLIENT_SECRET": "secret",
-    "API_URL": "https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002"
+    "API_URL": "https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002",
+    "OCC_PREFIX": "/rest/v2",
+    "BASE_SITE": "/electronics-spa",
+    "PREFIX_AND_BASESITE": "/rest/v2/electronics-spa",
+    "API_SERVER": "https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002/rest/v2",
+    "BASE_ENDPOINT": "https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002/rest/v2/electronics-spa"
```

Comment:
Duplicating API URL (and the prefix) accross many environment variables makes it harder to maintain. I'm not sure our releasing/deployment scripts will already pick it up too. Can we avoid those dupplications? I'd rather have a single source of truth.

Reply:
updated, fixed all routes to start with `occ_prefix` instead of a mix and match of `api_url` and `occ_prefix` as we only want to test for their endpoints for the response and not the whole url.

that was the main reason they had duplicates, but I normalized them